### ISSUE TITLE
NETCore 2.1+ - "Could not get any response" when Calling API secured with Certificate

### ISSUE THREAD

**Issue by skype007** [dotnet/runtime#33144]
We have a NETCore2.1+ API project with the following environments:
- Environment Development
- Environment DEMO
- Environment Quality (Secured with Certificate)
- Environement Staging (Secured with Certificate)

Using Postman, when I call a POST API with a JSON body (6263 lines according to Postman), the call works successfully in "Development" and "DEMO" environments. But not in "Quality" and "Staging" environments => meeting a "Could not get any response" in Postman with only this error in logs:

Error: socket hang up
Warning: This request did not get sent completely and might not have all the required system headers

If I reduce the JSON body (remove some elements in it --> let's say I cut it in half), the call works and if I try to send the full body (6263 lines of my JSON)...well it also works !!!

This issue occurs with NETCore2.1 but also in NETCore3.1 (I didn't try with lower versions)

**Comment by jeffschwMSFT** [comment]
@karelz

**Comment by karelz** [comment]
@skype007 did you try to create a minimal repro on your dev machine? Are you able to reproduce?
Or is it specific to your environments?

**Comment by davidsh** [comment]
>We have a NETCore2.1+ API project

Is this a .NET Core ASP.NET Core project? It sounds like it is and you are using POSTMAN as the client to test it?

If so, this issue would probably best belong in the dotnet/aspnetcore repo. Also, please attach a repro sample of your project so that it can be diagnosed.

**Comment by skype007** [comment]
@karelz @davidsh Yes this is a .NET Core ASP.NET Core project, the problem occurs when using HttpClient like usual (meeting TaskCanceledException) but I tried with Postman to see where the problem could be (if it's a problem with the code or a general problem).

Shall I copy the whole issue in "dotnet/aspnetcore" and close this issue ? If so, yes I can provide an example of the JSON ;)

**Comment by davidsh** [comment]
>Shall I copy the whole issue in "dotnet/aspnetcore" and close this issue ? If so, yes I can provide an example of the JSON ;)

We can move the issue for you.

@Tratcher - Can you move this to aspnetcore? This looks like a server-side issue of some kind.
